_rules/device-motion-user-interface-7677a9.md

```diff
@@ -30,17 +30,16 @@ The rule applies to any [HTML document][] with an associated [Window object][] t
 
 ## Expectation 1
 
-For each registered [device orientation event][device orientation] or [device motion event][device motion] in the test target, a set of one or more [instruments][instrument] is available in the same [web page][] supporting outcomes indistinguishable of those from the event.
-
-**Note:** The outcomes considered in this expectation are all the outcomes that result from the device motion operated functionality.
+For each registered [device orientation event][device orientation] or [device motion event][device motion] in the test target there exists a set of one or more [instruments][instrument] supporting indistinguishable results of those from the event.
 
 ## Expectation 2
 
-Each [instrument][] from expectation 1 is [visible][].
+For each [instrument][] in the set of [instruments][instrument] from Expectation 1, one of the following is true:
 
-## Expectation 3
+- the [instrument][] is in the same [web page][] of the test target; or
+- instructions for the user to locate the [instrument][] are in the same [web page][] of the test target.
```

Comment:
I don't think WCAG requires that there are instructions on every page.

Reply:
@WilcoFiers Yes. Same update resorting to #960 definition of 'clearly labeled location'